engine/admin/pruning.md

```diff
@@ -0,0 +1,187 @@
+---
+description: Pruning unused objects
+keywords: pruning, prune, images, volumes, containers, networks, disk, administration
+title: Prune unused Docker objects
+---
+
+Docker takes a conservative approach to cleaning up unused objects, such as
+images, containers, volumes, and networks: these objects are generally not
+removed unless you explicitly ask Docker to do so. This can cause Docker to use
+extra disk space. For each type of object, Docker provides a `prune` command.
+In addition, you can use `docker system prune` to clean up multiple types of
+objects at once. This topic shows how to use these `prune` commands.
+
+## Prune images
+
+The `docker image prune` command allows you to clean up unused images. By
+default, `docker image prune` only cleans up _dangling_ images. A dangling
+image is one that is not tagged and is not referenced by any running container.
+To remove dangling images:
+
+```bash
+$ docker image prune
+
+WARNING! This will remove all dangling images.
+Are you sure you want to continue? [y/N] y
+```
+
+To remove all images which are not used by existing containers, use the `-a`
+flag:
+
+```bash
+$ docker image prune -a
+
+WARNING! This will remove all images without at least one container associated to them.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+You can limit which images are pruned using filtering expressions with the
+`--filter` flag. For example, to only consider images created more than 24
+hours ago:
+
+```bash
+$ docker image prune -a --filter "until=24h"
+```
+
+Other filtering expressions are available. See the
+[`docker image prune` reference](/engine/reference/commandline/image_prune.md)
+for more examples.
+
+## Prune containers
+
+When you stop a container, it is not automatically removed unless you started
+it with the `--rm` flag. To see all containers on the Docker host, including
+stopped containers, use `docker ps -a`. You may be surprised how many containers
+exist, especially on a development system! A stopped container's read-only and
```

Comment:
the `read-only` layers are not removed (the read-only layers are what makes up an "image")